CHANGELOG.md

```diff
@@ -2,6 +2,16 @@
 
 All notable changes to this project will be documented in this file.
 
+## 0.3.1
```

Comment:
Do you plan to manually manage a changelog file? You may wish to update this, as I just created a v0.0.1 release, and when this PR is megred, it will create v0.1.0 release

Reply:
No, I prefer using git releases. I'll remove this file.